guides/sdk-labels.md

```diff
@@ -0,0 +1,37 @@
+# SDK's issues Labels
+
+⚠️ These labels are not the same as this repo's labels. They are meant to be used to tag the issues opened in the different SDK's repositories.
+
+We have many issues in every integration repository. These help us understand what each issue is about or its state.
+A clear label prevents us from re-reading the issue when we are scanning them, and they provide a context in the blink of an eye.
+
+Nonetheless, until now, every repo used its own label system. Thus, to avoid adapting to every repo's label choices, we want to create a consistent labels list to use.
+
+These are the labels that are encouraged to use.
+
+## Default labels
```

Comment:
I don't know if we should create two different sections @bidoubiwa they all have the same weight even if they were created by Github or by us.

Reply:
Okey! I hesitated so I don't have a real 100% opinion on the matter